yapapi/payload/vm.py

```diff
@@ -2,8 +2,14 @@
 from dataclasses import dataclass, field
 from enum import Enum
 import logging
-from typing import Optional
+import sys
+from typing import Optional, List
 from typing_extensions import Final
+if sys.version_info > (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
```

Comment:
Both `Final` and `Literal` are in `typing` since 3.8.
I have no opinion on whether we should always import them from `_extensions` or base import on `sys.version_info`, but I'm sure we should have them together.